### PR TITLE
feat(metadata): Display keywords and remarks

### LINKED ISSUE #707: Support viewing the metadata keywords property
### Description

As a catalog author, I want to view the keywords property in regard to metadata.

### Acceptance Criteria

* The `prop[@name='keyword']` field are properly displayed within the React library

### Proposed Solution

_No response_

### Additional Notes

- See https://pages.nist.gov/OSCAL/reference/latest/complete/json-reference/#/profile/metadata

### Implementation

- [ ] I can (or plan to) submit a pull request to implement this
- [ ] Implementing this may result in a breaking change

### LINKED ISSUE #710: Support viewing metadata `remarks`
### Description

As an OSCAL content author, I want to view `remarks` in regard to metadata.

### Acceptance Criteria

- [ ] The 'remarks' field is properly displayed within the React library with proper formatting

### Proposed Solution

This could be displayed at the bottom of the metadata section, underneath the "Parties" panel.

### Additional Notes

_No response_

### Implementation

- [ ] I can (or plan to) submit a pull request to implement this
- [ ] Implementing this may result in a breaking change

### PR DESCRIPTION
**Pull request by tuckerzp**
This adds the `OSCALMetadataKeywords` & `OSCALMetadataRemarks`
components.

closes #707
closes #710

### Testing

- http://localhost:3000/catalog/
- http://localhost:3000/catalog/?url=https://raw.githubusercontent.com/EasyDynamics/oscal-demo-content/test-content/catalogs/basic-test-catalog.json

image

### CODE REVIEW

File: src/test-data/CommonData.js [hunk 1]
```diff
@@ -123,6 +123,12 @@ export const metadataTestData = {
   roles: exampleRoles,
   locations: exampleLocations,
   version: "Revision 5",
+  props: [
+    {
+      name: "keywords",
+      value: "Keyword_1, Keyword_2,Keyword_3",
+    },
+  ],
 };
```
Comment (kylelaker) [inline src/test-data/CommonData.js:132]: Please also have tests for:

```json
{"props": []}
```

Displays nothing

```json
{
  "props": [
    {
      "name": "keywords",
      "value": "",
    }
  ]
}
```

Displays nothing

```json
{
  "props": [
    {
      "name": "keywords",
      "value": "Keyword_1",
    }
  ]
}
```

Displays 1 keyword

```json
{
  "props": [
    {
      "name": "keywords",
      "value": "Keyword_1",
      "ns": "https://test.easydynamics.com/ns/oscal"
    }
  ]
}
```

Displays nothing

File: src/components/OSCALMetadata.js [hunk 2]
```diff
@@ -610,6 +613,52 @@ function OSCALMetadataFieldArea(props) {
...
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <Button onClick={handleClick} variant="outlined" size="small">
+        Remarks
+      </Button>
```
[the remark this answers is not in the document]
Comment (tuckerzp) [reply src/components/OSCALMetadata.js]: That button has been removed.

**Review by kylelaker — APPROVED** [review]
This is terrific for initial viewing functionality. Thanks, @tuckerzp!

**Comment by Bronstrom** [comment]
Implementation reads nicely, tests and stories are great, and overall the content displays cleanly!

**Review by Bronstrom — APPROVED** [review]

**Pull request merged**